docs/dev/app/listings.md

```diff
@@ -1,9 +1,416 @@
+        // and returns the value of the New Field 
+    },
+];
+```
+
+**For a New Listing Type**
+
+- Create a a new array of fields of type `ListingField<NewListingType>[]`, where `NewListingType` is a placeholder for the actual listing type.  
+``` ts
+const newListingFields: ListingField<NewListingType>[] = [
+    // example field 
+    {
+        label: 'New Field',
+        getValue: data => data.newfield, 
+        // a function that takes data (type: NewListingType) 
+        // and returns the value of the New Field 
+    },
+    // ... other fields 
+];
+```
+- Add a type check to ensure that `listingFields` returns the fields for your desired listing. 
+``` tsx
+export default function ListingDetails( ... ) {
+    // ... 
+    const listingFields = useMemo(() => {
+        if (listingData.listing_type === 'NewListingType') {
+            return (
+                <ListingFields
+                    fields={newListingFields}
+                    listingData={listingData}
+                />
+            )
+        }
+        // ... handling of other listing types
+    }, [listingData, interpretation]);
+
+    // ... 
+}
+```
+
+## `ProfileMatch`
+
+::: tip Description
+For a logged in and onboarded user, `ProfileMatch` indicates aspects of their profile meet the requirements of a listing. For a not logged in and onboarded user, `ProfileMatch` will not render. 
+::: 
+
+- `matchIcon` conditionally renders the match icon based on the value of the argument, `match`, of type `boolean | undefined`. If `match` is undefined, the `grayDot` icon is returned, indicating that the listing does not include information about a given field. 
+- `renderIconGroup` renders the icon and text for a match-field. It takes in the argument `fields` of type `MatchField`. However, for fields that require the `yellowExclaimation` icon, `renderIconGroup` is not used. 
+    - The `yellowExclaimation` icon indicates that the user's data does not match a listing's details, but it is not a strict requirement. In particular, `yellowExclaimation` is used when an attorney does not match the language for the case.
+
+**ProfileMatch Props**
+
+- `listingData`: **`Listing`**
+
+    The listing whose ProfileMatch will be rendered. The rendering of fields is dependent on the `listing_type` of `listingData`. 
+
+- (optional) `interpretation`: **`boolean`**
+
+    Default value: false. Should be `true` for Langauge Support listings. Used to distinguish between Case Interpretations and Cases. 
+
+**Example Usage**
+
+- Langauge Support listings: 
+``` tsx:no-line-numbers
+<ProfileMatch listingData={listingData} interpretation />
+```
+- Non-Langauge Support listings: 
+``` tsx:no-line-numbers
+<ProfileMatch listingData={listingData} />
+```
+
+### How to Customize ProfileMatch Fields
+
+The easiest way to create a new profile match field is to create a new constant of type `MatchField` as defined below. 
+```ts
```

Comment:
could we remove line numbers for all the code blocks?

Reply:
sg, removed!